kindlecomicconverter/comic2ebook.py

```diff
@@ -1222,6 +1224,58 @@ def checkPre(source):
         raise UserWarning("Target directory is not writable.")
 
 
+def makeFusion(sources, qtgui=None):
+    filepath = []
+    GUI = qtgui
+    start = perf_counter()
+    pageTracker : int = 0
+    combinePath : str = os.path.join(os.path.dirname(sources[0]), "combinationTemp")
```

Comment:
Should name the final output the same name as the first file. Maybe append something at the ended like `fused`.